components/edit-collective/sections/SendingMoney.js

```diff
@@ -39,17 +77,58 @@ class SendingMoney extends React.Component {
             <SettingsSectionTitle>
               <FormattedMessage id="PayoutMethod.Type.Paypal" defaultMessage={'PayPal'} />
             </SettingsSectionTitle>
-            <P>
-              <FormattedMessage
-                id="collective.sendMoney.description"
-                defaultMessage={"PayPal is active by default. You don't have to configure anything."}
-              />
-            </P>
+            {!this.props.collective.settings?.disablePaypalPayouts && (
+              <P mb={3}>
+                <FormattedMessage
+                  id="collective.sendMoney.paypalEnabled.description"
+                  defaultMessage={
+                    'PayPal Payouts are active. Contributors can request Expenses to be paid with PayPal.'
+                  }
+                />
+              </P>
+            )}
+            {this.props.collective.settings?.disablePaypalPayouts && (
+              <P mb={3}>
+                <FormattedMessage
+                  id="collective.sendMoney.paypalDisabled.description"
+                  defaultMessage={
+                    'PayPal Payouts are disabled. Contributors are not able to request Expenses to be paid with PayPal.'
+                  }
+                />
+              </P>
+            )}
+            <StyledButton
+              loading={this.state.isSubmitting}
+              onClick={this.togglePaypal}
+              mt={2}
+              type="submit"
+              maxWidth={200}
+            >
+              {paypalConnectButton}
+            </StyledButton>
+            {this.state.error && (
+              <MessageBox type="error" withIcon my={3}>
+                {this.state.error}
+              </MessageBox>
+            )}
           </Fragment>
         )}
       </Fragment>
     );
   }
 }
 
-export default injectIntl(SendingMoney);
+const editCollectiveSettingsMutation = gql`
+  mutation EditCollectiveSettings($id: Int!, $settings: JSON) {
+    editCollective(collective: { id: $id, settings: $settings }) {
+      id
+      settings
+    }
+  }
+`;
+
+const addEditCollectiveSettingsMutation = graphql(editCollectiveSettingsMutation, {
```

Comment:
No need to iterate on this now, but for new developments using GraphQL V2 should be preferred. Here, we have a similar (and actually safer because it uses transactions) migration for this: `editAccountSetting`